sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubWebJobsBuilderExtensions.cs

```diff
@@ -84,26 +84,25 @@ public static IWebJobsBuilder AddEventHubs(this IWebJobsBuilder builder, Action<
 
         internal static void ConfigureInitialOffsetOptions(EventHubOptions options)
         {
-            string offsetType = options?.InitialOffsetOptions?.Type?.ToLower(CultureInfo.InvariantCulture) ?? string.Empty;
-            if (!string.IsNullOrEmpty(offsetType))
+            OffsetType? type = options?.InitialOffsetOptions?.Type;
+            if (type.HasValue)
             {
-                switch (offsetType)
+                switch (type)
                 {
-                    case "fromstart":
+                    case OffsetType.FromStart:
                         options.EventProcessorOptions.DefaultStartingPosition = EventPosition.Earliest;
                         break;
-                    case "fromend":
+                    case OffsetType.FromEnd:
                         options.EventProcessorOptions.DefaultStartingPosition = EventPosition.Latest;
                         break;
-                    case "fromenqueuedtime":
+                    case OffsetType.FromEnqueuedTime:
                         try
                         {
-                            DateTime enqueuedTimeUTC = DateTime.Parse(options.InitialOffsetOptions.EnqueuedTimeUTC, CultureInfo.InvariantCulture).ToUniversalTime();
-                            options.EventProcessorOptions.DefaultStartingPosition = EventPosition.FromEnqueuedTime(enqueuedTimeUTC);
+                            options.EventProcessorOptions.DefaultStartingPosition = EventPosition.FromEnqueuedTime(options.InitialOffsetOptions.EnqueuedTimeUtc.Value);
                         }
                         catch (FormatException fe)
                         {
-                            string message = $"{nameof(EventHubOptions)}:{nameof(InitialOffsetOptions)}:{nameof(InitialOffsetOptions.EnqueuedTimeUTC)} is configured with an invalid format. " +
+                            string message = $"{nameof(EventHubOptions)}:{nameof(InitialOffsetOptions)}:{nameof(InitialOffsetOptions.EnqueuedTimeUtc)} is configured with an invalid format. " +
                                 "Please use a format supported by DateTime.Parse().  e.g. 'yyyy-MM-ddTHH:mm:ssZ'";
```

Comment:
This message should be modified to reflect that we are now using System.DateTimeOffset instead of DateTime.Parse.

Reply:
The example timestamp might need to be modified to reflect the format used in tests: yyyy-MM-dd HH:mm:ssZ

Reply:
There is a test using the original test format as well - https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubConfigurationTests.cs#L132